adoc/reference-webui-systems.adoc

```diff
@@ -1458,7 +1459,59 @@ If you have System Group Administrator responsibilities assigned for your guest
 This is because it is possible to assign virtual guests on a single host to multiple System Group Administrators.
 Only users that have System Group Administrator privileges on the host system may create new virtual guests.
 
+For Salt systems, the btn:[Create Guest] button shows a dialog to configure and create a new virtual machine.
 
+[float]
+[[sdc-virt-host-guest-edit]]
+===== Editing a virtual machine
+
+[NOTE]
+====
+Traditional systems can only edit CPU and memory allocation.
+====
+
+The fields in this dialog are grouped into several panels.
+The [guimenu]``General`` panel contains the [guimenu]``CPU`` and [guimenu]``memory`` fields.
+The [guimenu]``Disks`` and [guimenu]``Network Interfaces`` panels list the fields corresponding to the matching devices of the virtal machine.
+The [guimenu]``Graphics`` panel allows configuring the display of the virtual machine.
+The [guimenu]``Schedule`` panel helps configuring when the edit should take place by choosing either an earliest time or an action chain to append to.
+
+[NOTE]
+====
+If a guest contains at least one disks of an unhandled type, any disk configuration editing will be prevented in order to avoid destroying that particular setup.
```

Comment:
Yes it is a disk (or network interface) type that is not recognised by SUMA. The definition of unhandled will evolve with versions since I will handle more types as the time goes.

Reply:
OK, so I would word that like this then:

`If a guest contains one or more disks or network interfaces not recognized by {susemgr}, you will not be able to edit the configuration.
This prevents any possibility of {susemgr} destroying the setup because of an unhandled type.`

That still gets the word 'unhandled' in there, but provides a definition. Overall, it should be clearer to the user what's actually happening.

Reply:
Ok, melded into the PR.